spec/rubocop/cop/style/multiline_list_spec.rb

```diff
@@ -0,0 +1,483 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::MultilineList, :config do
+  subject(:cop) { described_class.new(config) }
+
+  context 'method parameters' do
+    let(:cop_config) do
+      {
+        'EnableMethodParams' => true
+      }
+    end
+
+    context 'params listed on the first line' do
+      let(:source) do
+        ['def foo(bar,',
+         '  baz)',
+         '  do_something',
+         'end']
+      end
+
+      it 'detects the offense' do
+        inspect_source(cop, source)
+
+        expect(cop.offenses.length).to eq(1)
+        expect(cop.offenses.first.line).to eq(1)
+        expect(cop.highlights).to eq(['bar'])
+      end
+
+      it 'autocorrects the offense' do
+        new_source = autocorrect_source(cop, source)
+
+        expect(new_source).to eq(
```

Comment:
This doesn't seem like the correct auto-correction to me. The example given in the documentation comment shows
```ruby
# good with EnableMethodParams
def method(
  foo, bar,
  baz)
  do_something
end
```

The auto-correct places the first element left justified with the `def`. 
```ruby
def foo(
bar,
  baz)
  do_something
end
```

Should it be indented or is it relying on another cop to handle that indention?